Readme.md

```diff
@@ -28,3 +28,5 @@ To contribute examples, please fork this repository and add a self contained not
 1. Data available in the cloud (preferred): See `01_eccov4.ipynb` for example.
 2. Add files to the [xgcm-examples zenodo archive](https://zenodo.org/record/4421428#.X_XP7y1h3x9) and access them from within the notebook *(see `02_mitgcm.ipynb` and `04_nemo_idealized.ipynb` for examples). This should only be done for small datasets.
 3. Other way of accessing data online and download locally from the notebook: See `03_MOM6.ipynb` for examples.
+
+For more information on how to set up a development environment see the [xgcm contributors guide](https://xgcm.readthedocs.io/en/latest/contributor_guide.html).
```

Comment:
Maybe mention that even if these are 2 different git repositories, they use the same conda environment, which instructions are found in the [xgcm contributors guide](https://xgcm.readthedocs.io/en/latest/contributor_guide.html).
That was this point that was not clear for me.